tests/integration/kubernetes/k8s-confidential-attestation.bats

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bats
+# Copyright 2024 IBM Corporation
+# Copyright 2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/lib.sh"
+load "${BATS_TEST_DIRNAME}/confidential_common.sh"
+load "${BATS_TEST_DIRNAME}/confidential_kbs.sh"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+export KBS="${KBS:-false}"
+export test_key="aatest"
+export SUPPORTED_TEE_HYPERVISORS=("qemu-sev" "qemu-snp" "qemu-tdx" "qemu-se")
+export SUPPORTED_NON_TEE_HYPERVISORS=("qemu")
```

Comment:
`SUPPORTED_TEE_HYPERVISORS` and `SUPPORTED_NON_TEE_HYPERVISORS` are already defined in the `confidential_common.sh`. It's recommend to remove the lines.